projects/ebv-analyzer-app/src/app/ebv-selector/ebv-selector.component.ts

```diff
@@ -350,335 +243,51 @@ export class EbvSelectorComponent implements OnInit, OnDestroy {
-                mergeMap(([params, sessionToken]) =>
-                    this.http.get<EbvDataset>(`${this.config.API_URL}/ebv/dataset/${params.id}`, {
-                        headers: new HttpHeaders().set('Authorization', `Bearer ${sessionToken}`),
-                    }),
-                ),
+                first(),
+                mergeMap((params) => this.http.get<EbvDatasetResponse>(`https://portal.geobon.org/api/v1/datasets/${params.id}`)),
             )
-            .subscribe((dataset) => {
-                const ebvClass = this.ebvClasses.find((c) => c.name === dataset.ebvClass);
-
-                if (!ebvClass) {
-                    return;
+            .subscribe((response) => {
+                if (response.code !== 200) {
+                    this.notificationService.error('Could not load dataset');
                 }
 
-                this.ebvClass = ebvClass;
-                this.ebvNames = ebvClass.ebvNames;
-
-                this.ebvName = dataset.ebvName;
-
-                this.request<Array<EbvDataset>>(`ebv/datasets/${encodeURIComponent(dataset.ebvName)}`, undefined, (data) => {
-                    this.ebvDatasets = data;
-
-                    const selected = data.find((d) => d.id === dataset.id);
-
-                    if (!selected) {
-                        return;
-                    }
-
-                    this.setEbvDataset(selected, () => {
-                        this.selectDefaultGroupEntity();
-                        this.changeDetectorRef.markForCheck();
-                    });
-                });
-            });
-    }
-
-    private selectDefaultGroupEntity(): void {
-        if (!this.ebvTree) {
-            return;
-        }
-
-        let groups = this.ebvTree.tree.groups;
-        let index = 0;
-
-        while (groups.length > 0) {
-            this.setEbvPath(groups[0], index++);
-            groups = groups[0].groups;
-        }
-
-        const entity = this.ebvTree.tree.entities[0];
-        this.setEbvEntity(entity);
-
-        this.showEbv();
-    }
-
-    private generateGdalSourceNetCdfLayer(): Observable<RasterLayer> {
-        if (!this.ebvDataset || !this.ebvTree || !this.ebvDatasetId) {
-            throw Error('Missing dataset and loading info');
-        }
-
-        const workflow: WorkflowDict = {
-            type: 'Raster',
-            operator: {
-                type: 'GdalSource',
-                params: {
-                    data: {
-                        type: 'external',
-                        providerId: '1690c483-b17f-4d98-95c8-00a64849cd0b',
-                        layerId: JSON.stringify(this.ebvDatasetId),
-                    } as ExternalDataIdDict,
-                },
-            },
-        };
-
-        let colorizer = Colorizer.fromDict(this.ebvTree.tree.colorizer);
-
-        const currentGroup = this.ebvPath[this.ebvPath.length - 1];
-        if (currentGroup.dataRange) {
-            // rescale colors in colorizer
```

Comment:
The rescaling of colors does not work anymore.
When I create overviews, e.g., for dataset 25, I still get a range from 0 to 255 instead of 0 to 127.

Reply:
should this really be done everytime? Can't we compute the correct colorized during creation of the overviews?

Reply:
We can, but we have to do this in order to remove this here.

Reply:
so who shall fix it where now?

Reply:
Maybe start with "working code". So add it back here and we can refactor it later?

Reply:
as the colorizer is now built in the backend, I do the rescaling there now as well: 5cb092c17d1c92a6aea8f385207814d2d072525c